### PR TITLE
Update vat-calculator extension

### PR DESCRIPTION
**Pull request by Justiniscoding**
## Description

I added a command for quickly calculating the NET price of a value and also added the NET price to the main command so that the user can copy the net price or paste it to their currently active app

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/vat-calculator/src/noviewnet.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+export default async function quickVat(props) {
+  var number = getNetPrice(props.arguments.number);
+
+  if (getPreferenceValues().autopaste) {
+    showHUD(`Pasted Result to Active App`);
+    await runAppleScript(`
+      tell application "System Events"
+      keystroke "${number}" 
+      end tell
+      `);
```
Comment (pernielsentikaer) [inline extensions/vat-calculator/src/noviewnet.jsx:15]: Why not use [Clipboard.paste](https://developers.raycast.com/api-reference/clipboard#clipboard.paste) here?

File: extensions/vat-calculator/src/noviewnet.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Clipboard, showHUD, getPreferenceValues } from "@raycast/api";
+import { runAppleScript } from "@raycast/utils";
```
Comment (pernielsentikaer) [inline extensions/vat-calculator/src/noviewnet.jsx:2]:
```suggestion
```

File: extensions/vat-calculator/src/noviewnet.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+import { getNetPrice } from "./lib/vat";
+
+export default async function quickVat(props) {
+  var number = getNetPrice(props.arguments.number);
```
Comment (pernielsentikaer) [inline extensions/vat-calculator/src/noviewnet.jsx:7]: I would maybe add [CloseMainWindow](https://developers.raycast.com/api-reference/window-and-search-bar#closemainwindow) to ensure Raycast closes as soon as possible 🙂

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you 

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by Justiniscoding** [comment]
Thanks for the feedback, I've made the changes you requested.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**